short-response.md

```diff
@@ -121,10 +147,11 @@
   setLocation(address);
   console.log(`Our address is ${address}`);
   ```
-
+  - This logs `'829 Jefferson Ave.'` and `'Our address is 7 Marcus Garvey'` because when our function is invoked and given `address` only its value is passed to the function. Inside our function the parameter `location` is assigned this value, but reassiged to `'829 Jefferson Ave.'` and logged. Our `address` variable that is logged in the last line was not change because when we passed it to the function it was only by value and a copy was made for this purpose.
 
 13. **What do we mean when we say that JavaScript passes variables _by value_?**
-
+  
+    - This means the value of our variables is copied and pointed to by the variable that it was passed to. Our original variable still refereces the value it was assigned.
```

Comment:
# Question 13 Score:
## Technical
**3**: Correct!

## Writing
**2**: A typo ("refereces") but I'm docking you because your first sentence is a bit unclear.